funcx_sdk/funcx/serialize/concretes.py

```diff
@@ -134,6 +137,77 @@ def deserialize(self, payload):
         return function
 
 
+class CombinedCode(SerializeBase):
+    """This method uses multiple methods to serialize a function
+
+    # These should be a superset of the methods
+    Code from interpreter/main        : Yes
+    Code from notebooks               : Yes
+    Works with mismatching py versions: Yes
+    Decorated fns                     : Yes
+    """
+
+    identifier = "10\n"
+
+    def get_multiple_payloads(self, payload):
+        parts = self.chomp(payload).split(COMBINED_SEPARATOR)
+        assert len(parts) == 2 * len(DO_SERIALIZE_METHODS)
+        payloads = []
+        for index in range(int(len(parts) / 2)):
```

Comment:
The assert above has just guaranteed this is even and is the length of the `DO_SERIALIZE_METHODS`.  Could either do:
```python
for index in range(len(DO_SERIALIZE_METHODS)):
```
or
```python
for index in range(len(parts) // 2):
```

Reply:
Yes the 2nd option is easier to read, I agree